README.rst

```diff
@@ -37,7 +37,20 @@ framework. This project contains no plug-ins. You are free to use:
 Prerequisites
 -------------
 
-The supported versions of Python are Python >= 3.5.
+The supported versions of Python are Python >= 3.5.  Envisage requires:
 
 * `apptools <https://github.com/enthought/apptools>`_
 * `traits <https://github.com/enthought/traits>`_
+
+Envisage has the following optional dependencies:
+
+* `Ipykernel <https://github.com/ipython/ipykernel>`_
+* `Pyface <https://https://github.com/enthought/pyface>`_
+* `TraitsUI <https://https://github.com/enthought/traitsui>`_
+
+To build the full documentation one needs:
+
+* `Sphinx <https://pypi.org/project/Sphinx>`_ version 1.8 or later.
```

Comment:
I do not know for sure if version >=1.8 is specifically required.  Admittedly, I blindly took this from traits.

Reply:
Traits needs Sphinx >= 2.1; looks like the README for Traits is out of date there. I think the restriction on the Sphinx version for Traits resulted from the needs of the trait-documenter and the trait-documenter tests. It's *probably* safe to use an earlier version here, but 2.1 is a good minimum version.

Reply:
Created enthought/traits#1320 to fix the Traits README